experimental/nodejs/BUILD

```diff
@@ -2,12 +2,14 @@ package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 
+DISTRO_SUFFIXES = ("_debian9", "_debian10")
```

Comment:
You can load this from `distro.bzl`: https://github.com/GoogleContainerTools/distroless/blob/ad1103a45338e6c2ce08096f8a12c804d171af95/base/BUILD#L4

Reply:
Hey @chanseokoh I tried using `load("//base:distro.bzl", "DISTRO_SUFFIXES") `, but the travis builds fail with it while working locally.

However the tests pass without it: https://travis-ci.org/github/GoogleContainerTools/distroless/builds/722237561

Any pointers?

Btw: I added nodejs builds for both debian 9 and 10 and a default, which is still pointing to debian 9 like before.

Reply:
The build failed because of `BUILD` file formatting. You should remove the blank line shown below.
```diff
--- ./experimental/nodejs/BUILD	2020-08-29 07:34:11.000000000 +0000
+++ /tmp/buildifier-tmp-762079997	2020-08-29 07:34:17.217278010 +0000
@@ -1,7 +1,6 @@
 package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
-
 load("//base:distro.bzl", "DISTRO_SUFFIXES")
 
 NODEJS_MAJOR_VERISONS = ("10", "12", "14")
```
FYI, the failure log also tells you how you can automatically fix formatting.
```
Run 'buildifier -mode=fix $(find . -name 'BUILD*' -o -name 'WORKSPACE*' -o -name '*.bzl' -type f)' to fix formatting
```